### PR TITLE
Fix typos and grammar

### PR DESCRIPTION
**Pull request by hflynn**
Hopefully mostly fixed now, I just wasn't sure whether 'StrinDB' is supposed to be 'StringDB'?

### CODE REVIEW

**Comment by joshmoore** [comment]
`StringDB` yes.

File: README.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-- **IDR_API_example_script.ipynb**: Shows example of using the web API to extract metadata from the IDR.
+- **Getting_Started.ipynb**: How to connect, some simple data access
+- **Figure_1_Sampling_of_Phenotypes.ipynb**: Reproduces Fig. 1 of the paper: downloads annotations from all screens and computes and plots some statistics on phenotypes. Interative visualization is done using [bokeh](bokeh.pydata.org/)
+- **GeneNetwork.ipynb**: Reproduces Fig. 2 of the paper: downloads annotations from 3 screens with a phenotype in common, queries StringDB for interactions and plots the resulting network. Uses a conversion table for orthologues and gene identifiers which was built offline using biomart (see article for more details.). It uses [bokeh](bokeh.pydata.org/) and py2cytoscape to display networks.
+- **RohnPhenotypeClustering.ipynb**: Downloads annotations from IDR008, and performs some simple phenotypic clustering, building a figure similar to Fig. 1 of the corresponding paper. Builds a galery of thumbnails from images of several phenotypes.
```
Comment (mtbc) [inline README.md]: "gallery"

**Comment by hflynn** [comment]
Will push fixes for notebooks here too

**Comment by hflynn** [comment]
Right, hopefully that has caught everything! The images in those last couple of files crashed my text editor so I had to make the final edits in the gh interface.

**Comment by eleanorwilliams** [comment]
Where it says Figure 1b it should just say Figure 1. We don't have an 'a' and 'b' anymore.

File: notebooks/PCAanalysisOfCharmFeatures.ipynb [hunk 2]
```diff
@@ -580,7 +580,7 @@
     }
    ],
    "source": [
-    "# Visualisation of one stack. \n",
+    "# Visualization of one stack. \n",
     "# a Division is happening around t=220, and seem to have a clear signature, even with only 5 components\n",
```
Comment (eleanorwilliams) [inline notebooks/PCAanalysisOfCharmFeatures.ipynb]: change 'and seem to have a clear ' to 'and seems to have a clear '

**Comment by ac744** [comment]
when viewed/run in a notebook, the figure is slightly interactive (tooltips when hovering over points, zoom and pan, that sort of thing). Unfortunately it does not run when seen from the github interface (was that the question?)

**Comment by eleanorwilliams** [comment]
@ac744 It was really whether 'Interative' should be replaced by 'Interactive' or whether another word such as Iterative was meant.  But Interactive seems the most appropriate.   Should the word be 'Interactive'?

**Comment by ac744** [comment]
@eleanorwilliams oops, ok. Yes, interactive then :)
(my apologies for having such a bad spelling, didn't see it even when it's pointed out...)

File: notebooks/RohnPhenotypeClustering.ipynb [hunk 3]
```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Exemple of phenotypic clustering\n",
+    "# Example of phenotypic clustering\n",
     "\n",
-    "This notebook examplifies some steps involved in working with annotations from the IDR. It will build a figure similar to the figure 1.b of the paper from Rohn et al. \"Comparative RNAi screening identifies a conserved core metazoan actinome by phenotype\" (IDR008)."
+    "This notebook exemplifies some steps involved in working with annotations from the IDR. It will build a figure similar to the Figure 1.b of the paper from Rohn et al. \"Comparative RNAi screening identifies a conserved core metazoan actinome by phenotype\" (IDR008)."
```
Comment (mtbc) [inline notebooks/RohnPhenotypeClustering.ipynb]: another 1.b. here

Comment (eleanorwilliams) [inline notebooks/RohnPhenotypeClustering.ipynb]: I think this is referring to Figure 1.b in Rohn et al's paper though.  So its ok here.

Comment (hflynn) [inline notebooks/RohnPhenotypeClustering.ipynb]: Yeah, I did check that one but it's a different paper

**Comment by mtbc** [comment]
These changes look good to me, thank you @hflynn.

**Comment by hflynn** [comment]
That should be everything now @eleanorwilliams - thanks!

**Comment by eleanorwilliams** [comment]
All looks good to me @hflynn. Thanks.

**Comment by joshmoore** [comment]
Merging for inclusion into the already deployed prod36 (and the IDR-0.3.6) tag.

**Pull request merged**